clients/cmd/fluent-bit/loki.go

```diff
@@ -220,9 +221,21 @@ func removeKeys(records map[string]interface{}, keys []string) {
 	}
 }
 
-func createLine(records map[string]interface{}, f format) (string, error) {
+func (l *loki) createLine(records map[string]interface{}, f format) (string, error) {
 	switch f {
 	case jsonFormat:
+		for k, v := range records {
+			if s, ok := v.(string); ok && strings.Contains(s, "{") {
```

Comment:
What if the value is an array?

Reply:
We could do starts with `{` or `[`. What do you think?

Reply:
Yeah that sounds fair !